helm_deploy/hmpps-prisoner-to-nomis-update/values.yaml

```diff
@@ -34,6 +34,12 @@ generic-service:
     FEATURE_EVENT_PRISON_VISIT_REVISED: true
     FEATURE_EVENT_PRISON_VISIT_CANCELLED: true
     FEATURE_EVENT_INCENTIVES_IEP_REVIEW_INSERTED: true
+    FEATURE_EVENT_ACTIVITIES_ACTIVITY_SCHEDULE_CREATED: true
+    FEATURE_EVENT_ACTIVITIES_ACTIVITY_SCHEDULE_AMENDED: true
+    FEATURE_EVENT_ACTIVITIES_ACTIVITY_SCHEDULE_EXTENDED: true
+    FEATURE_EVENT_ACTIVITIES_ACTIVITY_AMENDED: true
+    FEATURE_EVENT_ACTIVITIES_PRISONER_ALLOCATED: true
+    FEATURE_EVENT_ACTIVITIES_PRISONER_DEALLOCATED: true
```

Comment:
This looks like what the schedule event names will be judging from 'activities.activity-schedule.created'